src/routes/web.php

```diff
@@ -11,4 +11,6 @@
 |
 */
 
-Route::view('/{any?}', 'app');
+Route::view('/', 'app');
```

Comment:
This change breaks Vue.js routing system... Please but it back.

Reply:
Fixed in 299808a